File: comu/com.energyict.comserver.comserver.core.extjs/src/main/web/js/mdc/src/commands/model/Command.js
/*
 * Copyright (c) 2017 by Honeywell International Inc. All Rights Reserved
 */
Ext.define('Mdc.commands.model.Command', {
    extend: 'Mdc.model.DeviceCommand',
    fields: [
        {
            name: 'deviceType',
            useNull: true,
            persist:false
        },
        {
            name:'deviceTypeAndConfiguration',
            persist: false,
            mapping: function (data) {
                var res = {};
                res.deviceType = data.deviceType;
                res.deviceConfiguration = data.deviceConfiguration;
                return res;
            }
        }
    ],

    proxy: {
        type: 'rest',
        url: '/api/ddr/devicemessages',

        reader: {
            type: 'json'
        }
    }
});

File: comu/com.energyict.mdc.processes.keyrenewal/src/main/java/com/energyict/mdc/processes/keyrenewal/api/impl/Command.java
/*
 * Copyright (c) 2018 by Honeywell International Inc. All Rights Reserved
 */

package com.energyict.mdc.processes.keyrenewal.api.impl;

public enum Command {
    RENEW_KEY,
    GENERATE_KEYPAIR,
    REQUEST_CSR,
    REQUEST_CERTIFICATE,
    UPLOAD_CERTIFICATE,
    TEST_COMMUNICATION;
}
